src/data/multiset/basic.lean

```diff
@@ -485,6 +485,8 @@ theorem card_eq_one {s : multiset α} : card s = 1 ↔ ∃ a, s = {a} :=
 theorem card_le_of_le {s t : multiset α} (h : s ≤ t) : card s ≤ card t :=
 le_induction_on h $ λ l₁ l₂, length_le_of_sublist
 
+alias card_le_of_le ← card_mono
```

Comment:
A separate lemma would be better here, as then you could make the type actually be `monotone`